Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Integral.lean

```diff
@@ -19,29 +20,45 @@ that the integral commutes with the continuous functional calculus under appropr
 + `cfc_integral`: given a function `f : X → 𝕜 → 𝕜`, we have that
   `cfc (fun r => ∫ x, f x r ∂μ) a = ∫ x, cfc (f x) a ∂μ`
   under appropriate conditions
++ `cfcₙ_integral`: given a function `f : X → 𝕜 → 𝕜`, we have that
+  `cfcₙ (fun r => ∫ x, f x r ∂μ) a = ∫ x, cfcₙ (f x) a ∂μ`
+  under appropriate conditions
 
 ## TODO
 
-+ Prove a similar result for the non-unital case
 + Lift this to the case where the CFC is over `ℝ≥0`
 + Use this to prove operator monotonicity and concavity/convexity of `rpow` and `log`
 -/
 
 open MeasureTheory
 
+open scoped ContinuousMapZero
+
+theorem ContinuousMapZero.integral_apply {X Y R : Type*} [MeasurableSpace X]
```

Comment:
I guess the point here is that `ContinuousMapZero` isn't imported by anything outside of the continuous functional calculus, so you can't put this earlier?

I *think* I would be in favor instead importing that into the file where `ContinuousMap.integral_apply` appears, as I don't believe it will increase the imports there too substantially (but maybe I'm wrong).

Reply:
Indeed that's why I had left it there, but now I checked and the extra transitive imports are literally just that one `ContinuousMapZero` file, so I moved the lemma.